src/MainWindow.vala

```diff
@@ -100,6 +100,19 @@ public class Dock.MainWindow : Gtk.ApplicationWindow {
     private static Wl.RegistryListener registry_listener;
     private void init_panel () {
         get_surface ().layout.connect_after (() => {
+            // manually set input region since container's shadow are is the content of the window
+            // and it still gets window events
+            unowned var surface = get_surface ();
+            var item_manager_width = ItemManager.get_default ().get_width ();
+            var shadow_size = (surface.width - item_manager_width) / 2;
+            var top_margin = 64 + shadow_size;
```

Comment:
Is there somewhere else this number is coming from? Can we make it a CONST or at least leave a comment about its origin?